service/src/test/java/uk/gov/hmcts/reform/fpl/service/OrganisationServiceTest.java

```diff
@@ -192,6 +201,43 @@ void shouldRethrowUnexpectedExceptions() {
         }
     }
 
+    @Nested
+    class FindManagedOrganisation {
+
+        @Test
+        void shouldFindOrganisationWhenUserRegisteredInOrganisation() {
+            when(organisationApi.findManagedUserOrganisation(null, SERVICE_AUTH_TOKEN,"ORGSA"))
```

Comment:
no space in front of "ORGSA". Please format code. It is not big deal, the problem is that if next dev touch this file it will start showing changes in places he or she did not touch. We use default Intellij formatter